task_templates/README.md

```diff
@@ -1,14 +1,55 @@
 # Templates for Custom Tasks
 
-In this directory are three folders - estimators, transforms, and pipelines
+In this directory there are three folders - estimators, transforms, and pipelines. 
+These are collectively known as "tasks". Each of these folders
+contain code snippets, i.e. tasks, that can be uploaded to DataRobot and incorporated into a blueprint. 
```

Comment:
I agree that it's a good idea to better explain how a task works, in that case we should elaborate a bit more though. Perhaps we can say something like this:
"DataRobot runs task's code as a part of the blueprint when it's being trained or used to score new data. To make sure DataRobot can correctly execute the code, the code must comply with a few requirements: (1) there must be a file custom.py or custom.R (depending on the language you use) and (2) custom.py/custom.R must contain one or multiple hooks (e.g. fit/score/transform/etc) that DataRobot will call and pass data to when a task is being trained or used for scoring. You can use multiple supporting files as well as needed, e.g. to clean up your code with helper functions. Once uploaded each tasks appears as a single box in the Blueprint. Please refer to Composable ML documentation to learn more about how tasks work."

Reply:
done!